client/src/components/NewVersionAvailable.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Dialog from './Dialog';
+import css from './NewVersionAvailable.css';
+
+const NewVersionAvailable = ({ newVersionAvailable }) => (
+  <Dialog
+    onClose={() => { window.location.href = '.'; }}
+    title="En ny versjon av applikasjonen er tilgjengelig"
```

Comment:
See 6ba07d39ebf896063de4d0a7f7c49b78d5cf805d. Also, see screenshot - there is spacing.

If this gets merged, #83 will be merged too, which removes the icon altogether.

What'cha think about "Applikasjonen har blitt oppdatert!"?